README.md

```diff
@@ -0,0 +1,51 @@
+# ansible-satellite6
+
+Ansible playbooks for Satellite 6 systems' management.
+
+## Setup
+
+### Prerequisities
+
+This repository will be Python3.6 based.
+
+You need Ansible installed. 
```

Comment:
Is there a minimum Ansible version?

Reply:
Currently, I am testing/working with 2.6, any suggestions as to which versions should we be supporting from? Currently, I think of from 2.5+ . Thoughts ?

Reply:
Maybe I should just test with Ansible 2.5 too and set that as the min version.

Reply:
This is still pending, will test and report here and then a re-review. Post which we will merge it.